filetypes/yaml.go

```diff
@@ -13,6 +14,17 @@ func FromYamlToMap(f backend.File) (map[string]interface{}, error) {
 		return nil, err
 	}
 
+	// Try to decrypt if the content is SOPS-encrypted
+	if sops.IsEncrypted(bytes) {
+		log.Debug().Str("file", f.Name()).Msg("attempting to decrypt SOPS-encrypted content")
+		decrypted, err := sops.DecryptYAML(bytes)
+		if err != nil {
+			log.Warn().Err(err).Str("file", f.Name()).Msg("failed to decrypt SOPS-encrypted content, using original content")
```

Comment:
Are you sure we should really swallow this error? Feels like we should down to the error on to the client, even if it's the fault of the configuration, for visibility & honesty.